simplewebrtc.js

```diff
@@ -434,8 +434,14 @@ SimpleWebRTC.prototype.stopScreenShare = function () {
 
 SimpleWebRTC.prototype.testReadiness = function () {
     var self = this;
-    if (this.webrtc.localStreams.length > 0 && this.sessionReady) {
-        self.emit('readyToCall', self.connection.getSessionid());
+    if (this.sessionReady) {
+        if (!this.config.media.video && !this.config.media.audio) {
+            self.emit('readyToCall', self.connection.getSessionid());
+        } else {
```

Comment:
I think you should be able to.  I was thinking logically when typing it up and did the if{}else{} then coded in the rest.  Logically should be equivalent though.